packages/calypso-e2e/src/hooks/screenshot.ts

```diff
@@ -16,6 +16,7 @@ import type { Context } from 'mocha';
  */
 export async function saveScreenshot( this: Context ): Promise< void > {
 	// If no test has been run, skip.
+	console.log( 'screenshot' );
```

Comment:
We run into a problem here where Jest doesn't seem to keep track of the test pass/fail status.